docs/deploy.md

```diff
@@ -0,0 +1,32 @@
+# Deploy Stargz Snapshotter
+
+We assume that you are using containerd (> v1.4.2) as a CRI runtime.
+
+## Systemd
+
+- Download release tarfile from https://github.com/containerd/stargz-snapshotter/releases/download/v0.5.0/stargz-snapshotter-v0.5.0-linux-amd64.tar.gz
+
+- Modify /etc/containerd/config.toml according to https://github.com/containerd/stargz-snapshotter/blob/master/script/config/etc/containerd/config.toml
+
+- Install fuse
+
+  ###### centos
+  ```
+  yum install fuse
+  modprobe fuse
+  ```
+
+  ###### ubunut
+  ```
+  apt-get install fuse
+  modprobe fuse
+  ```
+
+- Start stargz-snapshotter and restart containerd
+    ```
+    tar -xvf stargz-snapshotter-${version}-linux-${arch}.tar.gz -C /usr/local/bin
+    wget -O /etc/systemd/system/stargz-snapshotter.service https://github.com/containerd/stargz-snapshotter/blob/master/script/config/etc/systemd/system/stargz-snapshotter.service
```

Comment:
This doesn't seem work. Download the raw file.

```
wget -O /etc/systemd/system/stargz-snapshotter.service https://raw.githubusercontent.com/containerd/stargz-snapshotter/master/script/config/etc/systemd/system/stargz-snapshotter.service
```

Reply:
Have fix it.Thanks